docs/contribution-guide.md

```diff
@@ -1,19 +1,107 @@
+[apidoc]: https://kiltprotocol.github.io/sdk-js/api/index.html
+[jsdoc]: https://www.npmjs.com/package/eslint-plugin-jsdoc
+[typedoc]: https://github.com/TypeStrong/typedoc
+
 # Contribution Guide
 
-## NB
+## Documentation
+
+We want the KILT SDK to be easy to use for all fellow developers. We ❤️ well-documented code.
+
+KILT's API doc is available [here][apidoc].
+
+In the KILT SDK, modules and public methods are documented directly in the code, in what is called *docBlocks*. DocBlocks are special comments which document the code they precede. DocBlocks need to follow some linting rules. Make sure ESLint is activated in your code editor, so the errors and warnings are highlighted.
+
+We also check these rules on commit and in our CI pipeline, so that you're protected: you can't push a linting error to the repo.
+
+You don't need to - but if you're interested, you can see all the linting rules for the inline docBlocks in `.eslintrc.json`, and check what they mean on [jsdoc][jsdoc].
```

Comment:
Ok, let's clear this up:
[jsdoc](https://jsdoc.app/)  [(wiki)](https://en.wikipedia.org/wiki/JSDoc) is a spec and parser for javascript docblocks.
[tsdoc](https://github.com/microsoft/tsdoc) is the equivalent to that from microsoft for typescript and   
[eslint-plugin-jsdoc](https://www.npmjs.com/package/eslint-plugin-jsdoc) is an eslint plugin (normally for jsdoc) we use to lint our tsdoc docblock comments.

So to check, what the options mean, they have to look at `eslint-plugin-jsdoc` and not at `jsdoc`

Reply:
Got it, thanks. Fixed - I changed it to display eslint-plugin-jsdoc (the link is still right). The details you've mentioned are perfect for me to understand - however I left them out here since I feel contributors would only need to check eslint-plugin-jsdoc... what do you think?

Reply:
I think we should mention that we use tsdoc/jsdoc.
It's good, for people who know these and people who don't can just read further.

Reply:
Good idea - Updated!